utils/add_pack_patterns.py

```diff
@@ -0,0 +1,98 @@
+import lxml.etree as ET
+import argparse
+import re
+
+IOPAD_OLOGIC_REGEX = re.compile("OLOGICE3\.OQ_to_IOB33[MS]?\.O")
```

Comment:
Comment explaining what is going on here?

Reply:
Also this utility is in the generic folder, but seems xc7 specific?  Should be located in xc7/utils, or made general enough to be applicable to any arch?

Reply:
Done. I think this is going to be only xc7 specific. Being it a temporary patch, I would say that it shouldn't be used in the flow for the long-term.